config/local.yml

```diff
@@ -15,7 +15,9 @@ apps:
             strip_path_prefix: "/api/rhsm"
           - handler: rbac
             service_url: http://localhost:8001/api/rbac/v1
-            application: subscriptions
+            application:
```

Comment:
This is expecting a string, so this would need to be in the comma-separated format like `subscriptions,config-manager`